mevolib/fetch/BioSeqs.py

```diff
@@ -52,82 +28,60 @@
+ENTREZ_DB_DICT = {'nuccore': 'gbwithparts', 'nucest': 'gb', 'nucgss': 'gb', 'popset': 'gb', 'protein': 'gp'}
+# Constant that limits the maximum number of sequences that can be fetched
 MAX_NUM_SEQS = 100000
 
 
-def _get_entrez_db_rettype (entrez_db):
-    """
-    Check if the given entrez database is supported by the BioSeqs class.
-    Supported entrez databases are stored in 'ENTREZ_DB_DICT'.
-
-    Arguments:
-        entrez_db  (string)
-            Entrez database name.
+def _get_entrez_db_rettype (entrez_db: str) -> str:
+    """Checks if the given entrez database is supported by the BioSeqs class. Supported entrez databases 
+    are stored in 'ENTREZ_DB_DICT'.
+    Returns corresponding retrieval type for the given entrez database.
 
-    Returns:
-        string
-            Corresponding retrieval type for the given entrez database.
+    Args: 
+        entrez_db: Entrez database name
 
     Raises:
-        ValueError
-            If the introduced database is not supported.
+        ValueError: If the introduced database is not supported.
     """
-    if (entrez_db in ENTREZ_DB_DICT):
-        return (ENTREZ_DB_DICT[entrez_db])
+    if entrez_db in ENTREZ_DB_DICT:
+        return ENTREZ_DB_DICT[entrez_db]
     else:
-        message = '"{}" isn\'t a supported NCBI\'s Entrez DB'.format(entrez_db)
-        raise ValueError(message)
+        raise ValueError('"{}" isn\'t a supported NCBI\'s Entrez DB'.format(entrez_db))
 
 
+def _estimate_batch_size (record: str) -> int:
+    """Calculates the advisable batch size given the recommendations of Entrez and the record size.
 
-def _estimate_batch_size (record):
-    """
-    Arguments:
-        record  (string)
-            Record of the sequence in text format.
-
-    Returns:
-        int
-            Advisable batch size given the recommendations of Entrez and
-            the record size.
-
-    * It is calculated considering a 1Mbps download speed and a desirable
-    maximum fetch time of 5 minutes per batch.
+    Args:
+        record: Record of the sequence in text format.
+           
+    * It is calculated considering a 1Mbps download speed and a desirable maximum fetch time of 5 minutes 
+    per batch.
     """
     record_size = sys.getsizeof(record)
-    # Entrez advises a ceiling of 200 identifiers per request, and return at
-    # least 1 if the record is very big
+    # Entrez advises a ceiling of 200 identifiers per request, and return at least 1 if the record is very big
     return (int(max(min(200, numpy.floor(3.75e7/record_size)), 1)))
 
 
-#-------------------------------------------------------------------------------
-
 class BioSeqs:
-    """
-    A BioSeqs object holds SeqRecord objects (Bio.SeqRecord from Biopython) and
-    information about their source.
+    """A BioSeqs object holds SeqRecord objects (Bio.SeqRecord from Biopython) and information about their 
+    source.
```

Comment:
For "Attributes" you only report the public ones, following the same format as "Args" but with the type information, e.g.:
```
    Attributes:
        data (dict): Dictionary where the SeqRecord objects are stored.
```